src/Basement/CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ set(CMAKE_SKIP_RPATH ${FORTRAN_SKIP_RPATH})
 set(SKIP_BUILD_RPATH TRUE)
 
 add_executable(Basement ${SRC})
-target_compile_options(Basement PRIVATE -O1)
+target_compile_options(Basement PRIVATE $<$<CONFIG:Release>:-O1>)
```

Comment:
THIS is especially needed to get the segfault.

When building debug, do NOT ask for -O1 (the flags would be `-g -O1` which is weird)